generatebundlefile/promote.go

```diff
@@ -331,24 +317,22 @@ func (c *SDKClients) moveImage(image Image, authFile string, regionalBuildMode b
 		return fmt.Errorf("unable to copy image digest from source to destination repo: %w", err)
 	}
 
-	if regionalBuildMode {
-		destinationRegistryClient := c.ecrPublicClient
-		if c.ecrPublicClientRelease != nil {
-			destinationRegistryClient = c.ecrPublicClientRelease
-		}
-		destinationPrefix = fmt.Sprintf("docker://%s/%s", destinationRegistryClient.SourceRegistry, image.Repository)
-		BundleLog.Info(fmt.Sprintf("Promoting images to secondary ECR destination: %s/%s", destinationRegistryClient.SourceRegistry, image.Repository))
+	destinationRegistryClient := c.ecrPublicClient
```

Comment:
We actually don't need this secondary image part, we used to have this to push the packages images to the public ECR public.ecr.aws/eks-anywhere, but that didn't work after we changed some roles, and we are anyway handling it in the ECR replication

Reply:
Removed promotion to secondary ECR destination